### DISCUSSION TITLE
How to Ask for Help

### DISCUSSION THREAD

**Discussion by BryanNilsen** [nss-day-cohort-47/C47-Discussions#1] [Discuss a topic]
## RULES:

You have to answer **ALL** of the questions below to use the **Help Discussion**
**DO NOT ASK FOR HELP ON ANOTHER DISCUSSION TYPE**

---

- Your Name
- Have you searched the other help tickets? **If not, do that first** (Duplicates asks for help will be shamed ceaselessly)
  - If there are similar issues, make sure to give us their number/link to them and tell how yours is different

- Describe your issue
- List any error messages that appear
- What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
- What have you tried? Tell us everything you have tried.
- Copy/paste link **TO THE FILE** on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰

**NOTE:** We will not accept links to the whole repo. You must provide a link directly to the specific file

**ONCE YOU GET AN ANSWER:** mark the answer so that others can get the benefit of seeing the answer. It may very well be that you are asked to go back to your thread and update it with the answer if the instruction team helps you with your issue.

Either way, once you get a resolution either through the discussion or outside of it, you must update with the answer you get.

Did I mention **DO NOT ASK FOR HELP ON ANOTHER DISCUSSION TYPE**??

Give me a comment below letting us know that you understand these items or let us know in the comments if you have any questions.

**Comment by AutumnFoto** [comment]
👍

**Reply by BryanNilsen** [reply]
You win the points for reading the full discussion and letting us know you understand !!

**Reply by AutumnFoto** [reply]
Sorry it took me so long, finally posted my first question today lol! 💯
